vendor/assets/stylesheets/ombulabs/styleguide/1-utilities/_breakpoints.scss

```diff
@@ -0,0 +1,105 @@
+//
+//  MEDIA QUERIES
+//––––––––––––––––––––––––––––––––––––––––––––––––––
+
+// A map of breakpoints.
+$breakpoints: (
+  xs: 576px,
+  sm: 768px,
+  md: 992px,
+  lg: 1200px,
+  xlg: 1600px
+);
+
+
+//
+//  RESPOND ABOVE
+//––––––––––––––––––––––––––––––––––––––––––––––––––
+
+// @include respond-above(sm) {}
+@mixin respond-above($breakpoint) {
+
+  // If the breakpoint exists in the map.
+  @if map-has-key($breakpoints, $breakpoint) {
+
+    // Get the breakpoint value.
+    $breakpoint-value: map-get($breakpoints, $breakpoint);
+
+    // Write the media query.
+    @media (min-width: $breakpoint-value) {
+      @content;
+    }
+  
+  // If the breakpoint doesn't exist in the map.
+  } @else {
+
+    // Log a warning.
+    @warn 'Invalid breakpoint: #{$breakpoint}.';
+  }
+}
```

Comment:
@nykka I see that this utility mixing is not used anywhere. Could you remove it? 

I'm not sure this is a good idea. Usually we want to apply CSS rules from less specific (valid for every width) to more specific (media queries for narrow widths)

Reply:
@etagwerker I'm using the same breakpoints' file as we have in FastRuby. Do you want me to create a totally new breakpoints file or deleting those lines you mentioned above should be enough?

Reply:
@nykka Deleting should be enough.

Reply:
Done